src/main/asciidoc/java/index.adoc

```diff
@@ -1118,7 +1118,7 @@ dies. To create a _bare_ instance, launch:
 
 [source]
 ----
-vertx -ha
+vertx bare
```

Comment:
It didn't work in my trials.

Le mercredi 7 septembre 2016, Clement Escoffier <notifications@github.com>
a écrit :

> In src/main/asciidoc/java/index.adoc
> <https://github.com/eclipse/vert.x/pull/1617#discussion_r77815166>:
>
> > @@ -1118,7 +1118,7 @@ dies. To create a _bare_ instance, launch:
> >
> >  [source]
> >  ----
> > -vertx -ha
> > +vertx bare
>
> both should work no ?
> vertx -ha is the old way (but still supported)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/eclipse/vert.x/pull/1617/files/cbc258e675e295da52b3b5a926b98a3ab65dcb81#r77815166>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABlaTfn4cOgxaHfwiyj6opUpaoBrxrEks5qnrPggaJpZM4J206q>
> .
>